README.md

```diff
@@ -1,30 +1,18 @@
-Open Research?
+INTRODUCTION
 ==============
 
-We have released all our commercial HTM algorithm code to the open source 
-community within NuPIC. We use that code in our product. The NuPIC
-open source community and Numenta continues to maintain and improve that 
-regularly. 
+This repository contains the code for experimental algorithm work done internally at Numenta. A description of that research is [available here](https://www.numenta.com/neuroscience-research/).
 
-Internally though we continue to evolve and expand the ideas towards a 
-full blown cortical framework. Those research ideas are constantly in flux as
-we tweak and experiment. To go along with that we have a separate experimental 
-codebase that sits on top of NuPIC.   
+Open Research?
+==============
 
-We get a lot of questions about it. As such we wondered whether it is 
-possible to be even more open about that work.  Could we release our day 
-to day research code in a public repository? Would people get confused? Would
-it slow us down? 
+We have released all our commercial HTM algorithm code to the open source community within NuPIC. The NuPIC open source community continues to maintain and improve that regularly (see https://discourse.numenta.org for discussions on that codebase. Internally we continue to evolve our theory towards a full blown cortical framework.
 
-We discussed these tradeoffs on the NuPIC mailing list. Based on that 
-discussion, we decided to go ahead and create `nupic.research` It contains 
-the code for experimental algorithm work done internally at Numenta.
+We get a lot of questions about it and we wondered whether it is possible to be even more open about that work. Could we release our day to day research code in a public repository? Would people get confused? Would it slow us down?
 
-The code includes prototypes and experiments with different algorithm 
-implementations. This is all temporary, ever-changing experimental code, 
-which poses some challenges.
+We decided to go ahead and create htmresearch. It contains experimental algorithm code done internally at Numenta. The code includes prototypes and experiments with different algorithm implementations. It sits on top of NuPIC and requires you have NuPIC installed.
 
-Hence the following **DISCLAIMERS**:
+Our research ideas are constantly in flux as we tweak and experiment. This is all temporary, ever-changing experimental code, which poses some challenges. Hence the following DISCLAIMERS:
 
  
 What you should understand about this repository
```

Comment:
Could you change the itemized list to be this smaller list:

the code can change quickly and without warning as experiments are discarded and recreated
code will not be production-quality, buggy, or well documented
if we do work with external partners, that work will probably NOT be here
we might decide at some point to not do our research in the open anymore and instead delete the whole repository